src/MoBi.Presentation/Mappers/SpatialStructureToSpatialStructureDTOMapper.cs

```diff
@@ -24,6 +25,7 @@ public SpatialStructureDTO MapFrom(SpatialStructure spatialStructure)
          dto.TopContainer = spatialStructure.TopContainers.MapAllUsing(_containerToDTOContainerMapper);
          dto.Neighborhoods = _containerToDTOContainerMapper.MapFrom(spatialStructure.NeighborhoodsContainer);
          dto.MoleculeProperties = _containerToDTOContainerMapper.MapFrom(spatialStructure.GlobalMoleculeDependentProperties);
+         dto.Name = spatialStructure.DisplayName;
```

Comment:
This is the change for #1017. The ```dto.Name``` usually takes the ```objectBase.Name``` during mapping, but we are overriding that default in this mapper. This mapper is used whenever you need to select a spatial structure in a tree view from among all spatial structures in the building block repository.